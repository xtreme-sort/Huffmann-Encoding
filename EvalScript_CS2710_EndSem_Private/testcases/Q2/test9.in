DecodeMessage
((C,((((N,Z),(3,M)),((((8,1),R),n),(((((p,a),S),(A,((o,w),(6,H)))),W),(T,(m,(d,q)))))),(((v,(z,2)),h),((k,P),(F,_))))),((((X,((Q,I),u)),(0,9)),r),(D,e)));
001001001001001001110000011110010110011110101100110111001011111010101000101110001101011001111111001011111001011100110101111010011011101011000010010010110000101010101011001110011001110001001001110000011101111010111100111100111110110011010110011100111111011001010001011000100100100101110010110011000110010010110101011000101010010101001010110011110101101010110001011000010011011100011101010110011001000101100010001100001010001010000100010101010110010010111100111000101100100101101010001010110010010001011001001000110110110001010101111101000001011001111010110011000101111110101100111101011111001010010101111011010001000111000001011000010110011010111100101001011001101101010110101110001011010110011011001001011111101001101011010111100101111111110101110110110011110010001100010101011001100010110011000101100111001011000110001101011101000101110101010100000101100100111110111001100101101011000010011100000101100100101000110001110011101010111111010000010001100010001011101000100010111101001101000001011001110101010110001010011110010100110101011001001111110101111101010011100101100010101110011111000110101011001101010110000101101100000101101010111100111000111000111110101100111101010101011000000101100100100001100101100100101111001101001010101100011100100000110010010010010110011110101100010100110101000101111101001110101011001111100111010110010010110011000100111001101011001110010100110011111011111010001010100010110001000010001101011001100010111110100100111011000101100110101100001100110100000101100111001011111101011001100010110000010001110001110010010110000101100100110010100111000111001101000001010011000101100010001011111110001011000100010111001100111000100010001100100101001010100000101110010111111100011100100001111110011110010000110000100110100010111010101100010111100101100010110010100010110010010110011010110101100110100101010101000101011000101110001101010110001010100000111100101011111001111101010010111001010101111100101111101010101110010110011100101000001011001011001000001010000010001011010111100110110001011000001111010011010111111010111111010001010100010101111001010101100100101101010110011000100100101111011001010101010010001000001111010000010000010111111011100010110011011000110101100101001101110001011001111011110010001010110000001011001101011111011001101010001010100101010010100010101001010110010011101010110001010110011110101100110101011000101000101001001010010001010000010111111010110011010101100001010110001010001000111110101101010110001011111010111001110101100011101100001110011101011001100010000010111111100110101100001011001101010110001101011000101100100101111010001011110100110111001110101000001101100100101010101101010110000101011101001001011000010101100110110001110000010100001000101011111011001001011001110010111110011100100000100110101010111101110111100111110101111100101100011010110010010110101011001110010011011001010101100101110101100010101100001010111110010111110011100010111001011010101001100011010001111010001010111100101000101111001010011010101100100111000101111001000010001110001000101101110111010110011111000101010100010100001000100011001001011001111100110101110010111110011100010010100010001110101101100110111010101101011001001011001110010110011000111110101000101001110001010100000101111101100100000111000101100111101001001010101011111110001011001001011000101101010111110010111001011111001011100101111001011001101010110000011001101101100011010111100101100111010011010110111101011001111100001101010101001011000010100000110010100010110001101010001010110010010100000101111001011100100000101111110101101010110011000110010010111110100010101111101011100100000110000101000101011001100011010001010001010110000001011000010101100110110110001001000001110010001000110101111101101010110001010000111010001010110000001110101001001011001100010110010011000101010111010001110001110001001001101010001100110110101010000101010001011000010110011110101000001000101011101110110010100000101100110101101000101001010110011010101101011110011001001011111111101010001010110011100101011000110101100001010110011101000111000101010011011001111101011000101011000010101111110101100111111101100001011000111101100110110101011100101000010001101110101011000000100110101000001011111101011000000101100110001011001100010011011111010111111100000101010101110010110010010100101011000010101100111101100001011001110011001001011001100010111111010100000101110100011001000100010110011010110010010110011100101101100001000100010110011111001010000010110011010101100010101111010011101111010010100010101011001110010110011000101100000100010110011100011010011010111010010100010101000001011001111010110011100101001010110000000010110110101101010010010100101010000010110000001111001010101011001110011000110010110001010100101111101010101011001111100100101000001010010111100100110101000001001001011001101110010101011110010011010110011000101111101010110001000101010100001000001011000011000100010110010010111001011001110100011010010010110000101101010111100100100100001100110010001000100010100110010011000010110011100110101011000010101001100110101000101010001010110011110101101010110011100010010011000010110000001011001100010111111010110101110001011010001000101010010100100101111010011111011001110101011001001000101011000101011001101010101101101010110001010110110001000101100111110010010110011111010100100101100110110000100100101101010110011010110000111010101100111001110001011111011110001101100100101100010111110111010110000101100111001011100101100111001110000010111001111001000101011001111100010101111101001001010001010110011101000101011010101100101000100010110011100111110101100110001000001110110011010000010110001100100111110101000101011111001011001001101010110101011001001111001010000011100010110011000101100100100010101100111101110001010010101000001011000001010101000101011001101110011001001111001011001101111010000010110000101100111000100100010101010110001000101100110110000010110010100010101011000101100110101100011001010001010110001110101100001011100100000101111100110010010110010010011010100110001000101001010110001100011011100010110011110110010010110000101110001011111001011001100011000011000010010010110011010101100111001111101110011110001000101000001011001100010100000100111001001101010110000001111110001101011111010001000101000001010000010111100101110100010111001011001111100110101100000010110000010001101011111110001101000110011010010010110001011000010100001000110100110100011000101010111100111000110010010011010111111101010100001110110001001010101010101010101011001101011001010000010111110011001001010001100100101100000010110001010100010100110111110101111100101100000111010111001111001011110010100001011011001001001101011000000001011001111100101110110110011100111010111110111110110011010001010111110011001010001000100101001101111101100100111010101000001011000101010001100011010110011000101111010001101011000101100100101100110001011010100010100100110000111110101100101001101110101011000000110101011000011001010101100001001001010000010110001010100000100010101111001011001001001001010001010010100000101100100101111100101111100111010101110010100101011111001010101100001000001011011110110010010101010000010111110010110000101111010010010110000001100101000001111101011001101101010100010100000100110001110110000011110011010110101001010001011110101000101000001110101010101111001011000101011111010001101011001110100011101100100101111001010000010010011100011001101011100110000101100100101100100101101110011010100100111000101100111001101011111010110101110001100110110000101111110111110100110101000001011001110100100101010101100111101100110001011000000111110101111110101100101001101101101010110010010011010110101011001001011000101011000001001001011001101001000100101101100000100100101100010110011111010110010100011100111000110101111110101111001011001110010010011101010110001010100110110101011110010110011010110110101111101011001101011001110001000101100100101111001011001100010101010110011100101100110101011001101100010110101011111011010011010001010110001011110100011010010100110101101010000011001001110001111001001011110001101011110010101100000101100010111101001001011001100010110000001010000010110000101111110010001010101100001010110010010011010100010101001010110011111011000101011001010011010101010000010110011100100110111000100000100110100000101100101000111000100100010010000010110011110110011110100010001011010101100010111000101100110001000001011111001011111001011001110010110001010111100101100100101110011111011100010111100101111110111100110011010110101011111011001010000111010110001010111111011101010110011011000110101100000011001101010000010000010111111010100010111000110011100100111110110101100010101100100101111100101000101111110101011001101011001101011001111000101000001111111000010110011100111110101111110101100111001111001010000010100010110000101100000010101010110011110101111001100011101100001111001100110101100110001111110001001110101110010110101010010100001011001101001101111000010110001100000111110111110100000101100101000100100011010110011000101100111001000101011110010011010100011000110101111101011000100100001001110010010110011000101000101011001100010101010110011010111000101111100101100111110001010100000101100111101010101100010000010001010111100110011010110011000100010101100100110011100011011001101011011000111001101011001100100100111100101001010110001010111100101100110001011000101011111001010010110011100101000101010110101011001111110100010101100001011100101000110001000101100010100110111010110101111101011010101111001001001111101010000010110110001001000100010100000101100101000100010100010111011010101111001011001101010100110001001001100010110011110111100101110010001100010101100110101100111000010010011100010100010101000110010010110101011011110101100011000100010110101110001011111001100101100101000010101011000011001101110001100111000100010111100101101011010101100101000100010110001010011010110000001011001111010000011001010010100000101111100111000101110011101110100010101001010001111101110101011000010111010101000100010110000101011001110011000100101010110101101010110000101011001110010110110001000101100110001011000101011001101100010001100010101011001110010110011000101111110111110101111110101100111110010100100101001010110001010011011001101011001100010110011110101100010101100111011001011000010101111001111101110010101011100100011000011001011001110011010001111101011000000101100101000110101100000010111111101010010010110011110101111100110011011001110011010110000001001001011001001001010011010100000100110100111000110101000000101100011110011001001101100110101000110010010110000101011110100010101011010101100111101011000011110111100110010010111110010100101011001001011010101110100101000001011001101010111110010110011011100111110001100100111011000101010001010111001011001001001110001111001011111001100001011001110100110101000010000101100010101011001101010110011000101000010001000101001011111111011001001011001001011001110011111010110001111010110010010110011110101101010001010110000110000100010001010000010110001010110000111101010101110101000001011111001011000000101001010000100110111010001010010101110011001101011000101010000010110101011001001100110101001011110010011010010011001001011001100010000010000011001010001110001101001110000010001011001101110001011110100010101011001111011001001000101011001010010100000101100000000110011010000010111110010100010101100001110010100000101000000010010010000000101111100101000001011001100011001001011000000110011010011011000100011110011100100100100110100110100100101011000110101101100110101100111010001101111110000010111101010111010100110100000110010011100010110010010100010101100001010110011110001011111110010100100111010111111001001011111110010011001001010010101100110101110101000101011110011001101100111010101100001010110010010010011101010110011100111000110011010001011100010111110010110010011001100011111010111111111010110011110100010101100111001001110101011001110100010001011100101100111001110101001101100100101111010000100101010111110110101001001011001110010010010110000101010001010110001011001010001000101101100010001010011001010011010010011001101010001100010101011111010001110101110101000001011000101011011010110101100110101100111101111001010010101100100101000101101011111011000100100110110001000111110101000110101111010000011010100110101000001011111101011000010101111101110101100100100000101100110001110011101100110100111000001011000100010111001010001010000100000001101011100011000010011010110001011110011101010010010111111010110011110101100000100011100011010111110010100101010101011111101011000101011001010101011001100101010110010010110000001001101011011000101010011100100111010110101011001000010100101011000000101110100010101011001101100011010101010110011010101111100110001000110101110100010101011001100010111010001000101100110001001101110010001001000101011100000111010101100000010001010111100101100111101100001000110001001000101011000110010100101101010111011001010010110000101100111001011000010101100100110101011000101011000101011111001111101011001001011001110100010001111010101011000011010111111000100011001011110001101100110101100011001101011000101011100111100111111000010011011001000010000101010100101011100101100111101010010110101001001011001110010100000100010100100001011001001011001001100001011001100010111001011001110011001001011001111100100101001110100100100010101110101110011001101010010101111110110101111101000101011001111010100000101111010001101011011000001011111001011001110011111101010110011010110010101100010001110101011110011100010111101100101010101100010101111100100100101100110101100100101111010101011001110010111110010110011000101111001011001101010110000001010010110110001001011000101011000011111010111100110010010110001011010101110100111001101001010110101010000100010101110101011111001100100101100010001010101101100010110001000001001110010100010101011110010110000001011000010101000001011001101010010010111001011001101
Quit
