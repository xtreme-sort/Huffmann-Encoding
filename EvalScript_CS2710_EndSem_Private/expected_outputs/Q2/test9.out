C03MXFHwDdrZkh2eDdThF9Pv3an62Q9XPemF_26_r0IQ0TozWSRRHWSvMkPoIQv1NIn0mkAWZAZzZrrIdXHoqHdRmDQkCa2mR2hWkW2zqMWFqeTDDFZIoo6SuTIDr8A_kDWvMXA1u9rqNZQTQm9N6rSMDRrA_r_RDST_ChwahXWmkk_HnpANDAmhCrSDNz3HSM1_MrH9rzoM96R9e_Z1vXuod0PI9a9N6qopuu0azz9u9NRIQqIQT2QZ0R8Tqu0C_9DNDCMZPSFSzI0wh23r1SkhS8Fne9eRkrddrT68ADN8ZhFhIXF9qqZ1mnzWo3mDnRCNFXNqkwuA9kHFZpw_21RRZRAPSHwSZ3RC8qwaSZC_WS_TPvevkeoNq9a2nuSzmIeMke8h0nWaT0awuX8I_z6dkXMnFeF_dSrzW6MzrzeSaddkTWRuZeZm1FRrAkmNuQWDeHI1NQzH9Tdk3QPh9PWz6o_1MINdDNkH3nqI0ShdTdTmwpDhrum69WeHXDrRv8zIu1A8mTNqWozdI_TNv1ohC1pawrQXkQh_hSNeZpP3oAvrTuuQ919h8rZvH8ZTezXwhCRwWFzqe16nua6uIM2eSaqHevSe2hT8uPpM8qpooM_qXnTARaHv6zoq8TuCQwz6WXQH0XwSm9re3I6opI9uMT0INH6RpCWrh3R8pFn6vDSR_nH08RFM83wDnFModrvI8XaQAT6u3ahm3NDzCQR0v6haR911HW6C0vpoqWkWCZR3m9e2rAZSwnrrSWQH0Hr3wX0WwvP6kdeuzS_Pv6T6kCTFZHI_316IhAQ6_oNP9NS0_1dAhWAF8koAZHkR8pr1wDzFweNa2QInQwXAIS2S0IveakXdzAMRQRSukHzakdovv3w6_keQ8o8M0hp_udQ88mTIDH9ppuquZ93Sv8uMZImkzMqr8ev0rnnwzXdz10pS1M_dpeTFm8rr0MpCH0eh99r__2ZdzQ39_zP8S1uomuSzoWZ3v_A9Ppha0rv38S8ZmA313XAddPTRdnvNWeznNdaF0pzX_wr1NMCPXFhh3Ie1NPnFSdu6ur0m83k2TvAAWDh3k6h_Wk2vq_M860nH2Cp_qA9hrAMWASp0wCQrrX3S2eAu9uqm63PSRrrmwhr_w2QAmon6wwIrdD9ZSFu39WNzkF3eumnXSF0op8a_0CraAM1RHrIz9nN6MkNMNAuQQXH2DQWSkoNdd6SmAT_kmqF2WdD8eSqPwup28Nq1k20_hvrzd1_rw2HC8_DC6_q6F81vpnHmvevF2o_QeTWRNr9MFCSX__NAQuo6ZmM1udrQX90o1onwkdHINHnvXZm2oZA2u2Wu9o0FRSmoSdR20IWHDIvT1QSMPh_Wm3_8WQQ8AQ1PrmwRQ9CHFTZI26C3k110WWeSQWkdzD8ra9k2QmWhAQSMpHNz0XdkTPDI3uePaP1Ca6v0rhha6WQoSwQvr6oq_qH00RSM2oHS6Dam_krTZXD6hC_pAupqr3Hd229p3A398MMu8CruDzh910amIWHaeFzdRAWT0XwdwD_CzPIZTAMuDdv698XrIwo8QR_ezA6_SeAHWZaXQ8SaenPNdpRN9PCRT2S8WA2RFM3zoNNzuuMXZ2kmIHzZA0XpC2Nd1aD88C3NCd8ozp2MvuDk0MM3nuW96u_XmrPMNzkA1aHCq00P_0q0zRwPZm22raA3P6k2ZkdA2C_qeHZ6Mr6QT6PMzmX0r_h363a1SzQWQR09321IdurPNSWrh2HFRA1h_v0hQ_1rFXhM8qadeANoke2MXSCT1NXChkvMSFP3qHpuudRnqSArorApMWIM0PhACRpTIwunwdvuTIoTQoMkQIkCPpZmHvZQIvDRhTDRv6aAhSSd_A6QFrar_Qzeu2S9STF_X9zCNrRTHRh368Z3CAAvoT6zH0RD0ZTrD2Rqh_ZH8muWXd6_rwzrQPmkmDnSd3wzmr6domwpRhQrIv_mzShT993rr8IPdzSCnhIQCM0Impa8w3Tw
